src/components/views/EditMilestone.jsx

```diff
@@ -717,7 +717,7 @@ class EditMilestone extends Component {
         afterSave: (created, txUrl, res) => {
           if (created) {
             if (isProposed) {
-              const url = res ? `/campaigns/${res.campaign._id}/milestones/${res._id}` : undefined;
+              const url = res ? `/milestone/${res._slug}` : undefined;
```

Comment:
This change is not related to this issue, for each issue/task please fork a new branch from develop.